### ISSUE TITLE
Project is missing bunch of scripts

### ISSUE THREAD

**Issue by RonYanDaik** [pietru2004/Godot-WFC#2]
Missing bunch of scripts. At least that's what output says when trying to open 'main_panel.tscn'

 ```
Attempt to open script 'res://addons/WFC_Based_Gen/scripts/main_panel.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/main_panel.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/TileCreator.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/TileCreator.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/SwapArea.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/SwapArea.gd. Make sure resources have been imported by opening the project in the editor at least once.
  scene/resources/resource_format_text.cpp:283 - res://addons/WFC_Based_Gen/main_panel.tscn:33 - Parse Error: 
  Failed loading resource: res://addons/WFC_Based_Gen/main_panel.tscn. Make sure resources have been imported by opening the project in the editor at least once.

  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/main_panel.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/main_panel.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/TileCreator.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/TileCreator.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/SwapArea.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/SwapArea.gd. Make sure resources have been imported by opening the project in the editor at least once.
  scene/resources/resource_format_text.cpp:283 - res://addons/WFC_Based_Gen/main_panel.tscn:33 - Parse Error: 
  Failed loading resource: res://addons/WFC_Based_Gen/main_panel.tscn. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/main_panel.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/main_panel.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/TileCreator.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/TileCreator.gd. Make sure resources have been imported by opening the project in the editor at least once.
  Attempt to open script 'res://addons/WFC_Based_Gen/scripts/SwapArea.gd' resulted in error 'File not found'.
  Failed loading resource: res://addons/WFC_Based_Gen/scripts/SwapArea.gd. Make sure resources have been imported by opening the project in the editor at least once.
  scene/resources/resource_format_text.cpp:283 - res://addons/WFC_Based_Gen/main_panel.tscn:33 - Parse Error: 
  Failed loading resource: res://addons/WFC_Based_Gen/main_panel.tscn. Make sure resources have been imported by opening the project in the editor at least once.
```

**Comment by pietru2004** [comment]
Hm... that script should be removed... mayby I forgot to remove ref somewhere... Thanks for info...

**Comment by pietru2004** [comment]
Fixed in 3976d29278fef9f38eb2ce8f453814b5dce743ea - when I was recoding plugin I removed rule creation panel in favor of just using gridmap and forgot to remove this...

I will push update to godot asset library...
